### PR TITLE
Dark toggle theme button added

### PR DESCRIPTION
**Pull request by mahenoorsalat**
**Pull request : Add Toggle Button Functionality**

- Fixes issue #5 

- [x] Added a toggle button in the HTML that allows users to show or hide the content dynamically.
- [x] Created a .hidden CSS class to control the visibility of the content when the toggle button is clicked.
- [x] Updated the JavaScript to include an event listener for the toggle button, enabling the content to be hidden or displayed based on user interaction.

**Let me know if any other touch are needed! Thank you 👍**

Ping @sristy17

### CODE REVIEW

**Comment by sristy17** [comment]
> **Pull request : Add Toggle Button Functionality**
> 
> * Fixes issue [UI: Light/Dark Mode #5](https://github.com/sristy17/MindDrive/issues/5)
> * [x]  Added a toggle button in the HTML that allows users to show or hide the content dynamically.
> * [x]  Created a .hidden CSS class to control the visibility of the content when the toggle button is clicked.
> * [x]  Updated the JavaScript to include an event listener for the toggle button, enabling the content to be hidden or displayed based on user interaction.
> 
> **Let me know if any other touch are needed! Thank you 👍**
> 
> Ping @sristy17

Dark mode is not working in the home page of mobile version fix it please. Rest all looks fine

File: views/pages/about.ejs [hunk 1]
```diff
@@ -38,33 +63,44 @@
...
     <div class="text-sm items-center justify-center flex flex-col my-8">
-      Copyright © 2024. All rights reserved. 
+      Copyright © 2024. All rights reserved.
       <br>Sristy Paul & Pooranjoy Bhattacharya
     </div>
   </div>
+
   <%- include('../includes/navbar') %>
+
+  <script>
```
Comment (pooranjoyb) [inline views/pages/about.ejs]: Donot write scripts in the `views` files. We are maintaining a proper directory structure. Please write the scripts in `public/js` directory and link them in `views`

File: views/pages/index.ejs [hunk 2]
```diff
@@ -22,22 +22,69 @@
 
     <!-- Tailwind css cdn  -->
     <script src="https://cdn.tailwindcss.com"></script>
+
+    <style>
```
Comment (pooranjoyb) [inline views/pages/index.ejs]: Same thing please donot write styles in the `views` files. We are maintaining a proper directory structure. Please write the scripts in `public/css` directory and link them in `views`

File: views/pages/index.ejs [hunk 3]
```diff
@@ -226,13 +279,72 @@
...
+                    <br>Sristy Paul &amp; Pooranjoy Bhattacharya
                 </div>
 
                 <%- include('../includes/navbar') %>
 </body>
 <script src="/js/global.js"></script>
 
-</html>
\ No newline at end of file
+<script>
```
Comment (pooranjoyb) [inline views/pages/index.ejs]: should be in `public/js`

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: views/pages/index.ejs (hunk 3 above)
Comment (mahenoorsalat) [reply views/pages/index.ejs]: okay will  add there @pooranjoyb

**Comment by pooranjoyb** [comment]
also take a pull and resolve conflicts @mahenoorsalat

**Comment by pooranjoyb** [comment]
Please resolve conflicts 👍🏼  @mahenoorsalat

File: app.js
Comment (pooranjoyb) [inline app.js:1]: Why did you remove the database connection method? @mahenoorsalat

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

**Comment by sristy17** [comment]
Hey, just a small change 
Screenshot from 2024-10-03 19-12-02
Make the bg-colour of the home page shade lighter like the others. 

Screenshot from 2024-10-03 19-14-06
The bg-colour you applied for all looks good.

**Comment by pooranjoyb** [comment]
- Also make sure the cards of youtube are not in light mode, make them compatible to dark mode too* @mahenoorsalat

**Comment by pooranjoyb** [comment]
commits of this PR is also available in #39 right? @mahenoorsalat

**Comment by mahenoorsalat** [comment]
yeah it is! @pooranjoyb

**Comment by pooranjoyb** [comment]
this has a lotta conflicts, closing this one. Please create a new PR

**Pull request closed without merging**